README.md

```diff
@@ -11,36 +11,33 @@ description: "This sample shows how to use the Node.js V10 Storage SDK with read
 
 # Build a highly available app in Node.js with RA-GRS storage
 
-This sample shows how to use the Node.js V10 Storage SDK with read-access geo-redundant storage (RA-GRS) to create a highly available application that accesses files from secondary storage when there is a problem with primary storage, and then switches back when primary storage becomes available again. For more information, see [Designing HA Apps with RA-GRS storage](https://docs.microsoft.com/azure/storage/common/storage-designing-ha-apps-with-ragrs).
+This sample shows how to use the Node.js V10 Storage SDK with read-access geo-redundant storage (RA-GRS) to create a highly available app that accesses files from secondary storage when primary storage is down, and then switches back to primary storage when it becomes available again. For more information, see [Designing HA Apps with RA-GRS storage](https://docs.microsoft.com/azure/storage/common/storage-designing-ha-apps-with-ragrs).
 
-If you don't have a Microsoft Azure subscription, you can get a FREE trial account <a href="http://go.microsoft.com/fwlink/?LinkId=330212">here</a>.
+If you don't have a Microsoft Azure subscription, you can get a free trial account <a href="http://go.microsoft.com/fwlink/?LinkId=330212">here</a>.
 
-## How it works
+## SDK Versions
 
-This sample creates a new container in blob storage and uploads a sample file into it called `HelloWorld.jpg`. It then checks secondary storage repeatedly until both the container and file have been replicated. The user can then enter D to download the file or Q to quit. If the file is successfully downloaded, the sample indicates whether it came from primary or secondary storage. When the user quits the sample, it deletes the container and file. 
+In this sample, you will find the following folders:
 
-## How to run the sample
+* **[storage-node-ha-ra-grs-v10](./storage-node-ha-ra-grs-v10)** - references [Storage Blob SDK v10.3.0](https://www.npmjs.com/package/@azure/storage-blob/v/10.3.0)
+* **[storage-node-ha-ra-grs-v12](./storage-node-ha-ra-grs-v12)** - references [Storage Blob SDK v12.0.0](https://www.npmjs.com/package/@azure/storage-blob/v/12.0.0)
 
-To run this sample and verify that files are downloaded from primary storage:
+## Prerequisites
+
+Following prerequisites are needed to run this sample and verify that files are downloaded from primary storage:
 
 1. Add your storage account credentials to the `.env.example` file and then rename it to `.env`.
 
     ```
-    AZURE_STORAGE_ACCOUNT_NAME=<replace with your storage account name>
-    AZURE_STORAGE_ACCOUNT_ACCESS_KEY=<replace with your storage account access key>
+    AZURE_STORAGE_ACCOUNT_NAME=<Replace With Your Storage Account Name>
```

Comment:
In the last sample you used 'screaming caps' in the replacement token and here you use sentences.  Please be consistent across samples.